app/views/modules/_user_util_dropdowns.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if user_signed_in? %>
+    <div class="btn-group user_utility_links">
+      <a class="btn btn-default dropdown-toggle" data-toggle="dropdown" href="#"><span class="hidden-xs">&nbsp;<%= user_key %>&nbsp;</span><span class="sr-only">select for additional menu options</span><span class="caret"></span></a>
+      <ul class="dropdown-menu dropdown-menu-right" role="menu">
+        <li><%= link_to raw('<span class="glyphicon glyphicon-log-out"></span> log out'), destroy_user_session_path %></li>
```

Comment:
instead of rendering using `raw` and a string, make your `link_to` a block and you can pass in the markup you want instead.

```
<%= link_to  destroy_user_session_path do %>
  <span class="glyphicon glyphicon-log-out"></span>
  Log out
<% end %>
```

(I capitalised `log out` to `Log out` as well, to be consistent with `Login`)